tests/autoyast/installation.pm

```diff
@@ -118,6 +118,12 @@ sub run {
             $num_errors++;
         }
         elsif (match_has_tag('warning-pop-up')) {
+            # See poo#19978, no timeout on partition warning, hence need to click OK button to soft-fail
+            if (check_screen('warning-partition-reduced', 0)) {
+                record_soft_failure('bsc#1045470');
+                assert_and_click 'warning-popup-ok-button';
```

Comment:
just use a simple `send_key $cmd{ok};` No need for another needle. This would also allow to cover this in ncurses installation mode